buildapi_client/buildapi_client.py

```diff
@@ -52,6 +56,9 @@ def trigger_arbitrary_job(repo_name, builder, revision, auth, files=[], dry_run=
     )
     if req.status_code == 401:
         raise BuildapiAuthError("Your credentials were invalid. Please try again.")
+    
+    if files == [None, None]:
```

Comment:
Hi @mandaltapesh 
You can reply to my comments in context instead of at the bottom of the PR.

You can do this:
> if files is not None and any(files)

```
In [7]: a = None
In [6]: a is not None and any(a)
Out[6]: False
In [7]: a = []
In [8]: a is not None and any(a)
Out[8]: False
In [9]: a = [None]
In [10]: a is not None and any(a)
Out[10]: False
In [11]: a = [None, None]
In [12]: a is not None and any(a)
Out[12]: False
```